docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.9"
```

Comment:
A docker compose file can be very handy when your project uses localstack, postgres or any other dependencies.